packages/components/src/components/watermark/watermark.tsx

```diff
@@ -3,10 +3,11 @@ import './watermark.scss';
 
 type TProps = {
     image: CSSProperties['backgroundImage'];
+    opacity: CSSProperties['opacity'];
 };
 
-const Watermark: React.FC<React.PropsWithChildren<TProps>> = ({ image }) => (
-    <div className='watermark' style={{ backgroundImage: image }} />
+const Watermark: React.FC<React.PropsWithChildren<TProps>> = ({ image, opacity }) => (
```

Comment:
```suggestion
const Watermark: React.FC<React.PropsWithChildren<TProps>> = ({ image, opacity = 0.16 }) => (
```
@heorhi-deriv Or remove it from the CSS and give it a default here.